### ISSUE TITLE
Goland project names are not informative

### ISSUE THREAD

**Issue by bdleitner** [bazelbuild/intellij#3759]
I'm not sure if this is an issue with the bazel plugin or with Goland more generally (or both).  Unlike IJ, Goland doesn't seem to have a simple way to control a project's name (there is no name field in Project settings, for example).

In IJ, when I import a bazel project, I can name it and get descriptive names in the load window:
![IJ Good Project Names](https://user-images.githubusercontent.com/7515022/178160673-47155a7f-2386-4bf3-8ef9-190c41ad910d.png)

In Goland, on the other hand, all of the bazel projects use `.project-data-dir` and `.workspace` from Project Structure (note that there is no place to set the name):
![Goland Bad Project Settings](https://user-images.githubusercontent.com/7515022/178160690-4632d54c-5807-4b8e-8063-2e320c471fdc.png)

To give the following view when opening a project:
![Goland Bad Project Names](https://user-images.githubusercontent.com/7515022/178160703-5c5e21a7-18f3-41c6-95c7-a1bb7b654582.png)

All of the projects have the same name, so I have to look at internals of the path to figure out the right one, and this is doable but annoying.

**Comment by JamesRTaylor** [comment]
All projects in Open Recent menu list have the same name: `.project-data-dir, workspace.` Makes it very difficult to open the right project.

**Comment by sgowroji** [comment]
Hello @bdleitner, Could you please  confirm are you still seeing this with the latest release and also for non bazel projects ?

**Comment by JamesRTaylor** [comment]
I'm still seeing this with the latest bazel plugin, @sgowroji

**Comment by bdleitner** [comment]
> Hello @bdleitner, Could you please confirm are you still seeing this with the latest release and also for non bazel projects ?

Yes, I'm still seeing it in the latest version.  Non-bazel projects are not (and were never) affected.

**Comment by Vertexwahn** [comment]
I am having the same issue:
GoLand 2022.3.3 (on macOS M1), Plugin version: 2023.02.22.0.1-api-version-223

**Comment by Vertexwahn** [comment]
The issue still exists in:
GoLand 2023.2
Build #GO-232.8660.185, built on July 26, 2023
macOS 13.5

**Comment by JamesRTaylor** [comment]
It's crazy to me that the tooling for Bazel is so bad. It's been a year, and you can't fix a bug preventing the name of the project from appearing correctly? We've moved away from Bazel because of it's lack of usability and tooling (like many projects I've seen), so not interested anymore, but what is going on with this project?

**Comment by bdleitner** [comment]
@bazelbuild/triage not stale, still happening.

**Comment by Vertexwahn** [comment]
Still relevant!

**Comment by maratori** [comment]
Still relevant

**Comment by sudosaket** [comment]
Does this not happen on IntelliJ?

**Comment by bdleitner** [comment]
It has not in my experience.

**Comment by bdleitner** [comment]
This still happening.

Though it's less pressing.  The lack of attention to issue#4180 and issue#4356 have caused me to abandon bazel for go, and since I've found that bazel build on windows refuses to respect the CC environment variable and insists on using MSVC (which crashes trying to build the proto compiler) instead of GCC, so I might abandon bazel for my java work as well.

I'm extremely disappointed.

I also notice that in the options to comment, the only options are to Close, so I'm not optimistic that this will actually get unmarked as stale.

**Comment by odisseus** [comment]
You can rename an existing project manually by modifying the text file at `.ijwb/.idea/.name` and reopening it.

**Comment by odisseus** [comment]
Are these fields visible when importing a Bazel project in Goland?

<img width="700" height="367" alt="Image" src="https://github.com/user-attachments/assets/93ebefd3-164f-4fbb-b8df-058a9a49f595" />

**Comment by bdleitner** [comment]
Well, I doubt it's fixed, but since issue https://github.com/bazel-contrib/rules_go/issues/4356 is still open, I have abandoned the use of bazel for go.

**Comment by bdleitner** [comment]
I haven't checked it it's still an issue.  It's disappointing that this never gets any support, 
https://github.com/bazel-contrib/rules_go/issues/4356 is still a hard blocker on my coming back to bazel for go, which is disappointing, since I'm used to it (being a Googler).  The fact that no one is even assigned to that one is sad.
